src/plugins/intel_cpu/src/nodes/mkldnn_def_conv_node.cpp

```diff
@@ -674,9 +674,8 @@ void MKLDNNDeformableConvolutionNode::initSupportedPrimitiveDescriptors() {
 
     auto &weiDims = getInputShapeAtPort(WEI_ID).getDims();
     if (weiDims[1] == Shape::UNDEFINED_DIM || weiDims[0] == Shape::UNDEFINED_DIM ||
-        defConvAttr.group != 1 ||  // temporary workaround until jit impl. will correctly handle multigroup cases
-        (weiDims[1] % simd_w != 0)  // in_channels_per_gr !% simd_w
-        || ((weiDims[0] / defConvAttr.group) % simd_w != 0)) {  // out_channels_per_gr !% simd_w
+        (defConvAttr.group != 1 && ((weiDims[1] % simd_w != 0)  // in_channels_per_gr !% simd_w
```

Comment:
Why you removed comment about multigroup case? I though it is still valid.

Reply:
Because workaround consists in `AND` operation instead of `OR`. We have had to fallback temporary in every 'small channels' case, but now we need it only in multigroupped case, corresponding to limitations of algorithm.
Comment itself was necessary because of `weiDims[0] / defConvAttr.group` expression, regarding to `defConvAttr.group == 1` condition.

Reply:
The workaround consists of `OR` operation instead of `AND`. We have had to fallback in every 'small channels' case, now we do it only in multigroupped case, corresponding to li9mitations of algorithm. This comment itself was necessary because of `weiDims[0] / defConvAttr.group` expression, regarding `defConvAttr.group == 1` condition.